### ISSUE TITLE
Update to Gradle 7 to support build process using Java 11+

### ISSUE THREAD

**Issue by codeanticode** [processing/processing-android#661]
With the ECJ build with Processing 4, getting the error: "Could not run phased build action using connection to Gradle"

This is caused by the incompatibility between Gradle 6.x used by the mode and the Java 11 in the latest ECJ:

https://discuss.gradle.org/t/could-not-run-phased-build-action-using-connection-to-gradle/40466

**Comment by rupesh-kumar-lpu** [comment]
@codeanticode I saw you did some important changes in processing4 branch to make it compatible with p4. I compared these changes in processing4 branch with master branch. I tried processing 4 with [processing4 Android Mode Branch](https://github.com/processing/processing-android/tree/processing4), and I was able to compile and run apps on the device through that. Is this branch expected to be merged with Master in the future to support Android Mode in processing 4? Are there any known issues for this branch to work on? 
Thanks
